src/run/handlers/cache.cts

```diff
@@ -52,6 +52,27 @@ export class NetlifyCacheHandler implements CacheHandlerForMultipleVersions {
     return await encodeBlobKey(key)
   }
 
+  private getTTL(blob: NetlifyCacheHandlerValue) {
+    if (
+      blob.value?.kind === 'FETCH' ||
+      blob.value?.kind === 'ROUTE' ||
+      blob.value?.kind === 'APP_ROUTE' ||
+      blob.value?.kind === 'PAGE' ||
+      blob.value?.kind === 'PAGES' ||
+      blob.value?.kind === 'APP_PAGE'
+    ) {
```

Comment:
Is this list exhaustive?

Reply:
This is all `kind`s that we do support (and not hit this case https://github.com/opennextjs/opennextjs-netlify/blob/f3e24b1d2e4674574eef4c628d58b2d2a41e0be9/src/run/handlers/cache.cts#L285-L286

But right now this is manually constructed and there is no guard rail ensuring this list get's updated and in sync in the future.

Will think about best way of ensuring that all supported `kind`s are in here (but I would do this probably as follow up in case I don't figure out quick/easy solution)

Reply:
Ok, there is a way but it's rather not pretty - full `tsc` check results in pretty awful errors then that are really non-descript (if we miss kind) and the whole setup looks pretty iffy, so I would rather not go this road.

https://typescript-eslint.io/rules/switch-exhaustiveness-check/ would be great for this ... but it doesn't really work with version of eslint-typescript we have - we would need some major upgrades of this ... which would require major upgrades of bunch of other things. Those major upgrades also change default configs.

And lastly - we don't even control those versions ourselves - and shared config we use was last released ... over 2 years ago ( https://github.com/netlify/eslint-config-node/releases )

So, I won't be touching this in this PR and will keep it as is